src/omv/imlib/framebuffer.c

```diff
@@ -170,23 +170,20 @@ void framebuffer_update_jpeg_buffer() {
                 bool overflow = false;
                 if (0) {
                 #if OMV_RAW_PREVIEW_ENABLE
-                } else if (src->is_mutable &&
-                           image_size(src) <= OMV_JPEG_BUFFER_SIZE_MAX) {
+                } else if ((src->pixfmt == PIXFORMAT_GRAYSCALE) || (src->pixfmt == PIXFORMAT_RGB565)) {
```

Comment:
Please revert this and send a follow up fix for usbdbg if you like. Also you removed the check that raw image fits into the buffer.

Reply:
Okay, will get binary working.

Reply:
Actually may as well send it in the same PR.

Reply:
Done, tested. Binary streaming works with the IDE.

![image](https://github.com/user-attachments/assets/a9a517e5-6c3a-4ba4-a1e8-5acd7a3e73d1)

Reply:
No updates to the IDE are required. It's working already.